src/saga/disks.js

```diff
@@ -0,0 +1,101 @@
+import { takeEvery, put } from 'redux-saga/effects'
+
+import { CREATE_DISK_FOR_VM, REMOVE_DISK, EDIT_VM_DISK } from '../constants'
+import Api from 'ovirtapi'
+import { callExternalAction, delay } from './utils'
+import { fetchDisks } from '../sagas'
```

Comment:
I think better to move `fetchDisks` function to this file.

Reply:
I'm not going to start moving saga functions around at this time.  This **disks.js** is really just the sagas file from the legacy disk components moved here so I kept it distinct.

Sometime in the future we can refactor the sagas and split them up nicely.